index.html

```diff
@@ -191,9 +192,9 @@ <h2>Skills</h2>
     <section class="contact-me">
         <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
     <form action="https://formspree.io/f/mnqyglkn" method="post">
-        <input type="text"  id="name" name="user_name" placeholder="Full name" maxlength="30" required>
-        <input type="email"  id="email" name="use_email" placeholder="Email adress" required>
-        <textarea type="text" id="message" name="message" cols="30" rows="5" placeholder="Hey I'd like to connect to" maxlength="500" required></textarea>
+        <label for="name"><input type="text"  id="name" name="user_name" placeholder="Full name" maxlength="30" required></label>
+        <label for="email"><input type="email"  id="email" name="use_email" placeholder="Email adress" required></label>
+        <label for="message"><textarea type="text" id="message" name="message" cols="30" rows="5" placeholder="Hey I'd like to connect to" maxlength="500" required></textarea></label>
         <button type="submit">Get in Touch</button>   

```

Comment:
Kindly make sure there is no error in the contrast check. There is a small one with the buttons, kindly add different colors for the button states to increase accessibility. Here is the [result](https://www.aditus.io/button-contrast-checker/rawcdn-githack-com-2023-04-11-at-20-07-54-499?failing) of the error. Don't forget to add the change to the related section to your pull request description.
<img width="1194" alt="Screenshot 2023-04-11 at 11 33 12 PM" src="https://user-images.githubusercontent.com/25887253/231281722-41d72715-ab59-4764-8ec4-b8a9fbae1b52.png">

Reply:
-Added `background-color` to buttons on different states specifically when pressed to increase accessibility solving above raised bug.